test/framework/clusterstatevalidations/main_test.go

```diff
@@ -0,0 +1,94 @@
+package clusterstatevalidations_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/eks-anywhere/internal/test"
+	"github.com/aws/eks-anywhere/internal/test/envtest"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	clusterf "github.com/aws/eks-anywhere/test/framework/cluster"
+)
+
+var env *envtest.Environment
+
+const (
+	clusterName      = "test-cluster"
+	clusterNamespace = "test-namespace"
+)
+
+type clusterValidationTest struct {
```

Comment:
I would put this somewhere else and leave `main_test` only for the `TestMain` function